java/src/main/java/com/codurance/training/tasks/CheckOrUncheckService.java

```diff
@@ -0,0 +1,24 @@
+package com.codurance.training.tasks;
+
+import java.io.PrintWriter;
+import java.util.List;
+import java.util.Map;
+
+public interface CheckOrUncheckService {
+    default void setDone(String idString, boolean done, Map<String, List<Task>> tasks) {
+        int id = Integer.parseInt(idString);
+        for (Map.Entry<String, List<Task>> project : tasks.entrySet()) {
+            for (Task task : project.getValue()) {
+                if (task.getId() == id) {
+                    task.setDone(done);
+                    return;
+                }
+            }
+        }
+    }
```

Comment:
if possible remove default method and have this as private method or in utility 
as this will cause ambiguity in future